nes/src/cpu/instruction/ldy.rs

```diff
@@ -65,12 +67,28 @@ pub const ABSOLUTE_X: Instruction = Instruction {
 #[cfg(test)]
 mod tests {
   use super::*;
+  use cpu::instruction::into_byte;
+  use cpu::register::StatusFlags;
   use cpu::Registers;
 
   #[test]
   fn ldy_impl() {
     let mut core = Core::new(Registers::empty());
-    // TODO: test
+
+    ldy(&mut core, into_byte(42));
+    assert_eq!(core.reg.y_idx, into_byte(42));
+    assert!(!core.reg.status.contains(StatusFlags::N_FLAG));
+    assert!(!core.reg.status.contains(StatusFlags::Z_FLAG));
+
+    ldy(&mut core, into_byte(0));
+    assert_eq!(core.reg.y_idx, into_byte(0));
+    assert!(!core.reg.status.contains(StatusFlags::N_FLAG));
+    assert!(core.reg.status.contains(StatusFlags::Z_FLAG));
+
+    ldy(&mut core, into_byte(-28));
```

Comment:
Can we change `-28` to 128 which has the binary format `0b1000_0000`. Then we also don't need `into_byte`

Reply:
or I think you can do `-28i8 as u8`